src/crypto.ts

```diff
@@ -195,4 +200,41 @@ export default class Crypto {
       decodeBase64(formSecretKey)
     )
   }
+
+  /**
+   * Download and decrypt the given attachments.
+   * @param formSecretKey Secret key as a base-64 string
+   * @param decryptParams The params containing encrypted content and information.
+   * @returns The decrypted attachments if successful. Else, null will be returned.
+   * @throws {MissingPublicKeyError} if a public key is not provided when instantiating this class and is needed for verifying signed content.
+   */
+  downloadAndDecryptAttachments = async(formSecretKey: string, decryptParams: DecryptParams): Promise<DecryptedAttachments | null> => {
+    let decryptedRecords: DecryptedAttachments = {}
```

Comment:
as this is only assigned to in 1 place (line 235), would it be better expressed as a `map` to signify intent?

Reply:
I am thinking a `Record` is probably easier to use and is more "native" Javascript instead of an object?